Translation/TranslationManager.php

```diff
@@ -82,10 +82,14 @@ public function create(
             ->setOptions($options)
             ->setWorkTemplate($workTemplate)
             ->setCallback($this->generateProjectCallback())
-            ->setTextmasters($textmasters)
         ;
 
         $project->save();
+
+        if ($useMyTextmasters) {
+            $project->setTextmasters($project->getPotentialAuthors('my_textmaster'));
```

Comment:
Not sure it is gonna work as `getPotentialAuthors` returns an array of `AuthorInterface`